### ISSUE TITLE
PaperMC removal of CB package location error

### ISSUE THREAD

**Issue by zerolexi** [MiraculixxT/bluemap-marker#22]
[https://forums.papermc.io/threads/important-dev-psa-future-removal-of-cb-package-relocation.1106/](https://forums.papermc.io/threads/important-dev-psa-future-removal-of-cb-package-relocation.1106/l)

When using the plugin with PaperMC's test server jar, I get the following error:
```
[18:59:26] [Server thread/ERROR]: [BlueMap-Marker] Error initializing plugin 'bmm-paper-1.6.3.jar' in folder 'plugins\.paper-remapped' (Is it up to date?)
java.lang.NoClassDefFoundError: org/bukkit/craftbukkit/v1_20_R3/help/CustomHelpTopic
	at dev.jorel.commandapi.CommandAPIVersionHandler.getPlatform(CommandAPIVersionHandler.java:72) ~[?:?]
	at dev.jorel.commandapi.CommandAPI.onLoad(CommandAPI.java:103) ~[?:?]
	at de.miraculixx.bmm.BMMarker.onLoad(BMMarker.kt:25) ~[bmm-paper-1.6.3.jar:?]
	at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:59) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.storage.ServerPluginProviderStorage.processProvided(ServerPluginProviderStorage.java:18) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:39) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:36) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.craftbukkit.CraftServer.loadPlugins(CraftServer.java:507) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:274) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1133) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ClassNotFoundException: org.bukkit.craftbukkit.v1_20_R3.help.CustomHelpTopic
	at java.net.URLClassLoader.findClass(URLClassLoader.java:445) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:593) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	... 12 more
[18:59:28] [Server thread/ERROR]: Error occurred while enabling BlueMap-Marker v163 (Is it up to date?)
java.lang.IllegalStateException: Tried to access CommandAPIHandler instance, but it was null! Are you using CommandAPI features before calling CommandAPI#onLoad?
	at dev.jorel.commandapi.CommandAPIHandler.getInstance(CommandAPIHandler.java:169) ~[?:?]
	at dev.jorel.commandapi.CommandAPI.onEnable(CommandAPI.java:135) ~[?:?]
	at de.miraculixx.bmm.BMMarker.onEnable(BMMarker.kt:34) ~[bmm-paper-1.6.3.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:188) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:639) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:550) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:275) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1133) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
[18:59:28] [Server thread/INFO]: [BlueMap-Marker] Disabling BlueMap-Marker v163
[18:59:28] [Server thread/ERROR]: Error occurred while disabling BlueMap-Marker v163
kotlin.UninitializedPropertyAccessException: lateinit property blueMapInstance has not been initialized
	at de.miraculixx.bmm.BMMarker.onDisable(BMMarker.kt:38) ~[bmm-paper-1.6.3.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:290) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.disablePlugin(PaperPluginInstanceManager.java:223) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.disablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:537) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:192) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:639) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:550) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:275) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1133) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-"84a586f"]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

Not gonna pretend to know the implications of this, but I figured I'd submit an issue for it.

**Comment by Zoriot** [comment]
I think that can also be Closed because of  #24. The new Release works fine for Paper.

**Comment by MiraculixxT** [comment]
Fixed in newest (unreleased) version on `dev/2.0`. Will be on Modrinth in a few days
